.github/workflows/ci.yml

```diff
@@ -53,6 +53,34 @@ jobs:
           artifacts_path: dist
           anaconda_nightly_upload_token: ${{ secrets.UPLOAD_TOKEN }}
 
+      - name: Build a wheel and a sdist
+        run: |
+          # Bump version to avoid wheel name conflicts
+          sed -i 's/0.0.1/0.0.2/g' tests/test_package/pyproject.toml
+          rm ./dist/*
+          python -m build --outdir ./dist tests/test_package
```

Comment:
As attempting to apply multiple labels through repeated upload of the same package will fail as the package already exists, bump the version number of the test package to generate multiple packages that can be uploaded with different labels.